provider/fs/src/export/serializers/json.rs

```diff
@@ -86,25 +79,17 @@ impl AbstractSerializer for Serializer {
 }
 
 impl Serializer {
-    /// Creates a new serializer for [`serde_json`].
+    #[doc(hidden)]
     pub fn new(options: Options) -> Self {
         Self {
             style: options.style,
         }
     }
 
-    /// Convenience function to create a JSON serializer with the
-    /// [`StyleOption::Pretty`] format.
+    /// A JSON serializer that pretty-prints the output.
     pub fn pretty() -> Self {
```

Comment:
Issue: With all the APIs you're making hidden, it's not clear how people build one of these in non-pretty mode. It will appear in docs that `pretty()` is the only constructor. If someone finds `default()`, its behavior won't be documented. Please either un-hide these APIs or add a new constructor like `Json::compact()` to contrast with `Json::pretty()`